src/Domain/User/OrganizationUser.php

```diff
@@ -49,6 +49,11 @@ public function getRoles(): array
         return $this->roles;
     }
 
+    public function getRole(): ?string
+    {
+        return \count($this->roles) > 0 ? current($this->roles) : null;
```

Comment:
Est-ce que tu penses qu'à terme on devrait aussi changer la DB pour stocker un `role VARCHAR NOT NULL` ? Y a-t-il un obstacle à faire la migration dans cette PR ? (Pour la migration on pourrait assigner le rôle le plus "élevé") (On peut aussi la faire après cette PR, ce n'est pas bloquant)

Reply:
Je ne pense pas car ça nous laisse la possibilité d'en stocker plusieurs au besoin. On est encore en phase d'exploration sur cette feature, et on n'est pas à l'abri de se dire que finalement, il faudrait pouvoir en gérer plusieurs.